projects/fractally/packages/components/src/stories/SendDialog.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
SDField, in this case, is reusable. SendDialog is a concrete use case leveraging that reusable component. I think that's one criterion I'd use to differentiate between an Example and a component Story.

Also, because SDField is reusable, I could see us using this component in the Swap functionality too. And who knows where else. I commented elsewhere that I'd rather stay away from abbreviations in component names if we can help it (`SDField` doesn't tell me what it is unless I have more context.) But if we might end up using this in Swap functionality too, maybe the name of the component should be even _more_ generic than "SendDialogField."

Reply:
There is an issue with .mdx examples - Storybook inserts its own CSS into them that affects the look of our components, I've spent a lot of time trying to solve it but didn't find any solution yet. One possible way to solve this may be a custom Storybook theme. I will research this a little bit more later.

Reply:
I renamed SDField component. If we will make it reusable in other components, I'll list it as a story

Reply:
Sounds good. Thanks!